### PR TITLE
Add missing sdl keys

### PR DESCRIPTION
**Pull request by TheKeyblader**
# PR Details

Add missing key mapping for KeyboardSDL

## Related Issue

#1182 

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My change requires a change to the documentation.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by Eideren** [comment]
Thanks a bunch for looking into this ! 
This bit is slightly suspicious, maybe ``KKPPeriod`` maps to ``Keys.NumPadDecimal`` ? Haven't verified though.
```cs
map[KeyCode.KPeriod] = Keys.OemPeriod;
map[KeyCode.KKPPeriod] = Keys.OemPeriod;
```

**Comment by tebjan** [comment]
Lol, we scratched our heads about that one already... Thanks a lot!

**Comment by TheKeyblader** [comment]
I dont think mostly copied keymap from 
https://github.com/Rust-SDL2/rust-sdl2/blob/master/src/sdl2/keyboard/keycode.rs

File: sources/engine/Stride.Input/SDL/KeyboardSDL.cs [hunk 1]
```diff
@@ -232,7 +232,7 @@ private static class SDLKeys
                 map[KeyCode.KMinus] = Keys.OemMinus;
                 map[KeyCode.KKPMinus] = Keys.Subtract;
                 map[KeyCode.KDecimalseparator] = Keys.Decimal;
-                map[KeyCode.KKPDecimal] = Keys.Decimal;
+                map[KeyCode.KKPDecimal] = Keys.NumPadDecimal;
```
Comment (TheKeyblader) [inline sources/engine/Stride.Input/SDL/KeyboardSDL.cs:235]: Thx for this one :) @Eideren

Comment (Eideren) [reply sources/engine/Stride.Input/SDL/KeyboardSDL.cs:235]: Actually I looked around a bit and it's a bit more complicated than that, look here
http://kbdlayout.info/KBDBE/ <- Period numpad
http://kbdlayout.info/kbdbene <- Comma numpad
They should both map to ``NumPadDecimal`` then

Comment (TheKeyblader) [reply sources/engine/Stride.Input/SDL/KeyboardSDL.cs:235]: Agree 👍

**Comment by Eideren** [comment]
Thanks !

**Pull request merged**